enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1408 @@
+- Service Delivery should constrain permit & exclude configurations based on their internal policies. 
+  For example, customers may be forced to enable permissive windows which amount to at least 6 hours a month.
+
+### Implementation Details/Notes/Constraints
+
+#### ChangeManagement Stanza
+The `spec.changeManagement` stanza will be introduced into ClusterVersion and MachineConfigPool (for standalone profiles)
+and HostedCluster and NodePool (for HCP profiles).
+
+#### MaintenanceSchedule Strategy Configuration
+
+When a `[Hosted]ChangeManagementPolicy` is defined to use the `MaintenanceSchedule` strategy,
+a `maintenanceSchedule` stanza should also be provided to configure the strategy
+(if it is not, the policy is functionally identical to `Restrictive`).
+
+```yaml
+kind: ChanageManagementPolicy
+spec:
+  strategy: MaintenanceSchedule
+ 
+  maintenanceSchedule:
+
+    # Specifies a reoccurring permissive window. 
+    permit:  
+      
+      # If null, all dates are permitted and only exclude constrains permissive windows.
+      recurrence: 
+        # "frequency" is a discriminant for the unioned types which follow.
+        frequency: Yearly | Monthly | Weekly | Daily
+        
+        # Required when frequency=Daily
+        daily:
+          interval: <int> # How many days should pass before the next occurrence 0 < x < 731
+
+        # Required when frequency=Weekly
+        weekly:
+          daysOfWeek: 
+          - Monday
+          - Tuesday
+          - Wednesday
+          interval: <int> # How many weeks should pass before the next occurrence 0 < x < 27
+
+        # Required when frequency=Monthly
+        monthly:
+          # "by" is a discriminant for the unioned types which follow.
+          by: Day | Date
+          date:
+            datesOfMonth: 
+            - <int> # List of dates 0 < x < 32
+            interval: <int> # How many months should pass before the next occurrence 0 < x < 12
+          day:
+            days:
+            - weekOfMonth: First | Second | Third | Fourth | Fifth | Last
+              dayOfWeek: Monday | Tuesday | Wednesday | ... | Sunday
+            interval: <int> # How many months should pass before the next occurrence 0 < x < 12
+
+        # Required when frequency=Yearly
+        yearly:
+          # "by" is a discriminant for the unioned types which follow.
+          by: Day | Date
+          date:
+            datesOfMonth: 
+            - <int> # List of dates 0 < x < 32
+            month: January | February | ... | December
+          day:
+            days:
+            - weekOfMonth: First | Second | Third | Fourth | Fifth | Last
+              dayOfWeek: Monday | Tuesday | Wednesday | ... | Sunday
+            month: January | February | ... | December
+      
+      # Given the identification of a date by a recurrence rule, at what time (always UTC) can the 
+      # permissive window begin. "00:00" if unset.
+      startTime: <time-of-day|null>

```

Comment:
We will validate that this is a valid time in hours and minutes? We can use a regex for this I think